cmd/pebble/cmd_validate.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2014-2020 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/canonical/pebble/internal/plan"
+	"github.com/jessevdk/go-flags"
+)
+
+var shortValidateHelp = "Validate daemon configration"
+var longValidateHelp = `
+Perform validation of daemon configuration files and exit.
```

Comment:
Fixed type. Ben I think it would be very helpful for me if we could have a quick brainstorm on your second proposal. I think it makes a lot of sense, so much so that I wonder if we should not add the option "globally" so the $PEBBLE directory can be specified from command-line. But there seems to be a bit of a startup-race between needing the socket path and resolving the command-line options.

Reply:
Agreed we would split this out and see if an option such as "--pebble-dir" could be added globally to also allow the daemon to get a custom pebble dir from commandline.

Reply:
@flotter and I discussed this on a call, and agreed that we should leave this PR as is (using the env var) but consider adding a `--pebble-dir` (and possibly `--pebble-socket`) command line argument globally, but do that in a separate PR.